app/package.json

```diff
@@ -57,6 +57,7 @@
     "@credo-ts/question-answer": "0.5.13",
     "@credo-ts/react-hooks": "0.6.0",
     "@credo-ts/react-native": "0.5.13",
+    "@credo-ts/webvh": "file:../credo-ts/packages/webvh",
```

Comment:
a file dependency wont work. Is that because we are waiting for a new published package from from credo?